### PR TITLE
fix: Fix docs error

### PR DESCRIPTION
**Pull request by zzxming**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our Commit Message Guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #58 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

syntax 模块在模块中使用需要手动传递 highlight.js

## Summary by CodeRabbit

- **New Features**
	- Enhanced syntax highlighting functionality in the rich text editor with more detailed configuration options.
  
- **Improvements**
	- Transitioned to a modular approach for syntax highlighting, improving the editor’s capability to handle code snippets.

### CODE REVIEW

File: packages/docs/fluent-editor/demos/get-html.vue [hunk 1]
```diff
@@ -45,10 +44,9 @@ onMounted(() => {
       theme: 'snow',
       modules: {
         'toolbar': TOOLBAR_CONFIG,
-        'syntax': true,
+        'syntax': { hljs },
         'emoji-toolbar': true,
         'file': true,
-        'formula': true,
```
Comment (kagol) [inline packages/docs/fluent-editor/demos/get-html.vue:51]: @zzxming 这个demo主要演示 HTML 的预览效果，所以需要有尽可能全的功能，插入公式的功能不能删除哦

**Comment by kagol** [comment]
@zzxming PR标题请使用：`fix: xxx` / `feat: xx`等格式，这样会有机器人自动打 label。

File: packages/docs/fluent-editor/demos/get-html.vue (hunk 1 above)
Comment (zzxming) [reply packages/docs/fluent-editor/demos/get-html.vue:51]: 公式是 format 不是 module，依赖 window.katex ，不需要 module 配置

Comment (kagol) [reply packages/docs/fluent-editor/demos/get-html.vue:51]: > 公式是 format 不是 module，依赖 window.katex ，不需要 module 配置

对哦，那demo描述也得改下。

**Pull request merged**